.eslintrc

```diff
@@ -1,4 +1,7 @@
 {
     "extends": ["@yandex-cloud/eslint-config", "@yandex-cloud/eslint-config/prettier"],
-    "root": true
+    "root": true,
+    "env": {
+        "node": true
```

Comment:
This is necessary in order not to include the package @yandex-cloud/eslint-config/server